mmv1/third_party/terraform/.teamcity/components/inputs/services_ga.kt

```diff
@@ -141,6 +141,11 @@ var ServicesListGa = mapOf(
         "displayName" to "Certificatemanager",
         "path" to "./google/services/certificatemanager"
     ),
+    "chronicle" to mapOf(
+        "name" to "chronicle",
+        "displayName" to "Chronicle",
+        "path" to "./google-beta/services/chronicle"
+    ),
```

Comment:
chronicle is not needed to be added to ga services, as it is beta only now.

Reply:
Actually I tried that, but the presubmits are failing if the changes are not done to the ga services.